demo-app/src/main/java/com/gravatar/demoapp/MainActivity.kt

```diff
@@ -13,7 +13,7 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
 
         // Initialize the Gravatar SDK with the API key if it is available
-        BuildConfig.DEMO_GRAVATAR_API_KEY?.let { Gravatar.initialize(it) }
+        BuildConfig.DEMO_GRAVATAR_API_KEY?.let { Gravatar.apiKey(it).context(this) }
```

Comment:
```suggestion
        BuildConfig.DEMO_GRAVATAR_API_KEY?.let { Gravatar.apiKey(it).context(applicationContext) }
```

I guess it works, but I think this should be the application context, no?

Reply:
I would say it should work with any context. But let's change it to prevent issues.